dudect/src/main.rs

```diff
@@ -0,0 +1,52 @@
+//! Fuzz tests for `crypto-bigint`.
+//!
+//! These use the `dudect_bencher` crate to check certain operations for
+//! constant-time behavior.
+
+use crypto_bigint::{Limb, UInt, U256};
+use dudect_bencher::{ctbench_main, ctbench_main_with_seeds, BenchRng, Class, CtRunner};
+use rand::{Rng, RngCore};
+
+/// Generate a random `UInt`.
+// TODO(tarcieri): use `UInt::random` when `dudect_bencher` updates `rand`
```

Comment:
@rozbb if you have a moment, it'd be nice to get the `rand` version bumped

Reply:
Just published v0.5. Let me know if it works for you.

Reply:
Seems good!